watchman.sh

```diff
@@ -252,7 +252,7 @@ $inotify_cmd | while read key; do
         _status="$?"
 
         if [ "$_status" == "0" ]; then
-            success "Success!" >&2
+            success "Success!" >&1
```

Comment:
The reason the output for Success goes to `stderr` is because it isn't relevant to business logic and is primarily for logging. I have observed that logs going to `stderr` and business logic to `stdout` is a fairly common practice. Moreover, since `watchman` runs another command, I don't want to mess with the `stdout` output.

Reply:
Ahh okay got your point! I am fairly new to scripting so this was nice to know..thanks 👍🏼